packages/components/src/DataTable/Footer.css

```diff
@@ -0,0 +1,37 @@
+.footerContainer {
+  display: flex;
+  align-items: center;
```

Comment:
Similar comment as in the main CSS file, we should be using Atlantis tokens throughout and avoiding magic numbers (52, 70, etc)